backend/src/exercise/http-handler/secure-http.ts

```diff
@@ -0,0 +1,54 @@
+// TODO: How bad is that?
+// eslint-disable-next-line @typescript-eslint/no-shadow
+import type { Express, Request } from 'express';
```

Comment:
```suggestion
import type { Express, Request as ExpressRequest } from 'express';
```
And rename `Request` to `ExpressRequest` in the rest of the file.